# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1264676655 18000
#      Thu Jan 28 06:04:15 2010 -0500
# Node ID 57e24c53e063d18467dfb0c20d407bed57b023f7
# Parent  f7ba6cfe7fb74c2266530945ca8316aae7bb0032
doc/interpreter/obsolete.texi: new file; list removed functions

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,15 @@
+2010-01-28  John W. Eaton  <jwe@octave.org>
+
+	* interpreter/obsolete.txi: New file.
+	* interpreter/Makefile.am (MUNGED_TEXI_SRC): Add it to the list.
+	* interpreter/octave.texi (Obsolete Functions): New menu item.
+	Include obsolete.texi.
+
 2010-01-27  Jaroslav Hajek  <highegg@gmail.com>
 
 	* interpreter/expr.txi: Document using dummy outputs.
 	* interpreter/func.txi: Document using dummy inputs.
 
 2010-01-20  John W. Eaton  <jwe@octave.org>
 
 	* interpreter/munge-texi.cc: Delete Emacs local-variable settings.
diff --git a/doc/interpreter/Makefile.am b/doc/interpreter/Makefile.am
--- a/doc/interpreter/Makefile.am
+++ b/doc/interpreter/Makefile.am
@@ -114,16 +114,17 @@ MUNGED_TEXI_SRC = \
   install.texi \
   interp.texi \
   intro.texi \
   io.texi \
   linalg.texi \
   matrix.texi \
   nonlin.texi \
   numbers.texi \
+  obsolete.texi \
   oop.texi \
   op-idx.texi \
   optim.texi \
   package.texi \
   plot.texi \
   poly.texi \
   preface.texi \
   quad.texi \
diff --git a/doc/interpreter/octave.texi b/doc/interpreter/octave.texi
--- a/doc/interpreter/octave.texi
+++ b/doc/interpreter/octave.texi
@@ -182,16 +182,17 @@ It corresponds to GNU Octave version @va
 * Audio Processing::            
 * Object Oriented Programming::            
 * System Utilities::            
 * Packages:: 
 * Dynamically Linked Functions::
 * Test and Demo Functions::
 * Tips and Standards::                        
 * Contributing Guidelines::
+* Obsolete Functions::
 * Trouble::                     If you have trouble installing Octave.
 * Installation::                How to configure, compile and install Octave.
 * Emacs Octave Support::                       
 @c * Grammar::                     
 * Copying::                     The GNU General Public License.
 * Concept Index::               An item for each concept.
 * Function Index::              An item for each documented function.
 * Operator Index::              An item for each documented operator.
@@ -858,16 +859,17 @@ Emacs Octave Support
 
 @c ------------------------------------------------------------------------
 @c Appendices start here. 
 
 @include dynamic.texi
 @include testfun.texi
 @include tips.texi
 @include contrib.texi
+@include obsolete.texi
 @include bugs.texi
 @include install.texi
 @include emacs.texi
 @c @include grammar.texi
 @include gpl.texi
 
 @c ------------------------------------------------------------------------
 @c Indices start here. 
